components/chat/chatBar/search/catalog.tsx

```diff
@@ -0,0 +1,355 @@
+    },
+    {
+      name: 'Crawl Website',
+      description: 'Allows the assistant to search a website.',
+      url: 'github.com/gptscript-ai/search-website',
+      tags: ['search', 'web', 'site'],
+      icon: <GoSearch className="text-lg" />,
+    },
+    {
+      name: 'Browser',
+      description:
+        'Provides the assistant with the ability to interact with the web via a Chrome window.',
+      url: 'github.com/gptscript-ai/browser',
+      tags: ['browser', 'web', 'chrome', 'search'],
+      icon: <GoBrowser className="text-lg" />,
+    },
+  ],
+  Productivity: [
+    {
+      name: 'Slack',
+      description: 'Allows the assistant to interact with Slack.',
+      url: 'github.com/gptscript-ai/tools/apis/slack/write',
+      tags: ['slack', 'messaging', 'teams', 'api'],
+      icon: <AiOutlineSlack className="text-lg" />,
+    },
+    {
+      name: 'Notion',
+      description: 'Allows the assistant to interact with Notion.',
+      url: 'github.com/gptscript-ai/tools/apis/notion/write',
+      tags: ['notion', 'documentation', 'notes', 'api'],
+      icon: <SiNotion className="text-lg" />,
+    },
+    {
+      name: 'Hubspot',
+      description: 'Allows the assistant to interact with Hubspot.',
+      url: 'github.com/gptscript-ai/tools/apis/hubspot/crm',
+      tags: ['hubspot', 'api'],
+      icon: <FaHubspot className="text-lg" />,
+    },
+    {
+      name: 'Outlook Mail',
+      description:
+        'Allows the assistant to send and receive emails via Outlook.',
+      url: 'github.com/gptscript-ai/tools/apis/outlook/mail/manage',
+      tags: ['email', 'office', 'microsoft', 'service'],
+      icon: <PiMicrosoftOutlookLogoDuotone className="text-lg" />,
+    },
+    {
+      name: 'Outlook Calendar',
+      description: 'Allows the assistant to interact with Outlook Calendar.',
+      url: 'github.com/gptscript-ai/tools/apis/outlook/calendar/manage',
+      tags: ['calendar', 'office', 'microsoft', 'service'],
+      icon: <PiMicrosoftOutlookLogoDuotone className="text-lg" />,
+    },
+  ],
+  'Working with Local Files': [
+    {
+      name: 'Structured Data Querier',
+      description: 'Query Excel spreadsheets, CSV files, and JSON files.',
+      url: 'github.com/gptscript-ai/structured-data-querier',
+      tags: ['data', 'structured', 'query'],
+      icon: <PiMicrosoftExcelLogo className="text-lg" />,
+    },
+    {
+      name: 'JSON Query',
+      description: 'Allows the assistant to query JSON data.',
+      url: 'github.com/gptscript-ai/json-query',
+      tags: ['json', 'query', 'data'],
+      icon: <SiJson className="text-lg" />,
+    },
+    {
+      name: 'Workspace',
+      description:
+        'Allows the assistant to be aware of and iteract with files in your workspace.',
+      url: 'github.com/gptscript-ai/context/workspace',
+      tags: ['workspace', 'files'],
+      icon: <GoFileDirectory className="text-lg" />,
+    },
+  ],
+  'Coding and DevOps': [
+    {
+      name: 'Github',
```

Comment:
@njhale just finished getting a big overhaul to this in. the github entru should now be
```
url: 'github.com/gptscript-ai/tools/apis/github/write',
tags: ['github', 'api'],
```

can address this in this pr or an immediate follow up